kwok/toolchain.dev.openshift.com_usersignups.yaml

```diff
@@ -0,0 +1,166 @@
+---
```

Comment:
While having a copy of the file makes things simple for now, we need to think how do we keep it up to date.

Reply:
Keeping this yaml up to date can be automated.